core-developers/become-core-developer.rst

```diff
@@ -25,37 +25,39 @@ an official offer. How core developers come to that agreement are outlined in
 Gaining Commit Privileges
 =========================
 
-The steps to gaining commit privileges are:
+After a candidate has demonstrated consistent contributions, commit privileges
+are granted through these steps:
 
-1. A core developer starts a poll in the
-   `Committers category`_ on the `Python Discourse`_ (``discuss.python.org``)
+1. A core developer (submitter, usually the mentor) starts a poll in the
+   `Committers category`_ on the `Python Discourse`_ (``discuss.python.org``).
```

Comment:
People often confuse Discourse with Discord, so I think the URL is there to prevent/mitigate confusion.

Reply:
Yeah, that's why I kept it there, to avoid any ambiguity and follow convention elsewhere. But we could unlink `Python Discourse`, since that URL is arguably somewhat redundant given the category was already linked directly; I only added it originally since I already had the link target defined and that was how it was presented elsewhere.